assets/js/cache-warmup.js

```diff
@@ -375,6 +375,7 @@
                             // send request
                             return $.ajax({
                                     url: url.absolute,
```

Comment:
ok super. hast du mal versucht ob man im gleichen browser, während ein einem anderen tab dieses re-indexing läuft redaxo im backend noch bedienbar ist?

ich vermute dass aktuell weitere tabs blockiert (oder zumindest stark beeinträchtigt) werden, durch ein parallel laufendes indexing (voraussgesetzt man ist im gleichen browser und in der gleichen session)